xform-marc21-to-xml/src/main/java/edu/stanford/AuthDBProperties.java

```diff
@@ -76,6 +80,37 @@ private void initDataSourceProperties(Properties properties) {
         this.userPass = properties.getProperty("PASS");
     }
 
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
I need some explanation about what these new blocks are supposed to do (`equals` and `hashCode`).